docs/actions/match.md

```diff
@@ -161,6 +161,23 @@ end
 
 Then all your team has to do is `fastlane certificates` and keys, certs and profiles for all targets will be synced.
 
+#### Handle multiple apps per developer/distribution certificate
+
+If you want to use single developer and distribution certificate for multiple apps belonging to the same development team, you may use the same signing identities repo and branch to store signing identities for your apps:
+
+Matchfile for App #1:
+```no-highlight
+git_url "https://github.com/example/example-repo.git"
+git_branch "master"
+```
+
+Matchfile for App #2:
+```no-highlight
+git_url "https://github.com/example/example-repo.git"
+git_branch "master"
+```
+Match will reuse developer/distribution certificate and will create separate provisioning profiles for each app.
```

Comment:
Match => _match_ (with underscores for italic)

Reply:
... reuse developer/distrivution certificate ... => ... resue the certificate ...

Reply:
Done